Makefile

```diff
@@ -12,21 +12,8 @@ $(DCURL_LIB): $(DCURL_DIR)
 	@echo
 	$(info Modify $^/build/local.mk for your environments.)
 	$(MAKE) -C $^ all
-
-TESTS += $(wildcard tests/*.py)
-TESTS += $(wildcard tests/tangleid/*.sh)
-
-check: server.py $(DCURL_LIB)
-	@ TMP_PID=`mktemp /tmp/server_pid.XXXXXX`; \
-	echo "Running test suite..." ; \
-	( python $^ & echo $$! > $${TMP_PID} ); \
-	sleep 3 ; \
-	for i in $(TESTS); do \
-	    ( echo "\n\n==[ $$i ]==\n"; $$i || kill -9 `cat $${TMP_PID}` ) \
-	done ; \
-	kill -9 `cat $${TMP_PID}`
-	@$(RM) $${TMP_PID}
-
+check:
```

Comment:
Ensure the dependency for build procedure.

Reply:
Replace it with "check: server.py $(DCURL_LIB)". 👍